src/sidekick/module.js

```diff
@@ -896,6 +908,14 @@
         action: async (evt) => {
           const { status } = sk;
           const updatePreview = async (ranBefore) => {
+            if (status.edit.sourceLocation.startsWith('gdrive:') && !isSupportedGoogleType(status.edit.contentType)) {
```

Comment:
Since it's only used in a single place, do we really need a function for it?
```suggestion
            if (status.edit.sourceLocation.startsWith('gdrive:')
              && contentType !== 'application/vnd.google-apps.document'
              && contentType !== 'application/vnd.google-apps.spreadsheet') {
```

Reply:
My thoughts were to decouple this in case other content types need to be added.

Reply:
I see. Nothing planned for now. We can still optimize when that becomes the case.